.circleci/config.yml

```diff
@@ -57,27 +57,29 @@ only_dev: &only_dev
 jobs:
   apollo:
     docker:
-      - image: circleci/node:16.13.1@sha256:1803e9ed7deec9456ad2609124b7333d40b2eec0cf34998ae766cbf90c9a3625
+      - image: cimg/base:2023.04
```

Comment:
a more minimal image since we don't need node for this step

Reply:
Rover just invokes the managed federation steps we have in apollo studio -- I don't think the OS should matter as long as the tool works